packages/website/plugins/generated-rule-docs/insertions/insertNewRuleReferences.ts

```diff
@@ -147,3 +161,46 @@ export async function insertNewRuleReferences(
 
   return eslintrc;
 }
+
+function linkToConfigs(configs: string[]): mdast.Node[] {
+  const links = configs.map(
+    (config): mdast.Link => ({
+      children: [
+        {
+          type: 'inlineCode',
+          value: config,
+        } as mdast.InlineCode,
+      ],
+      type: 'link',
+      url: `/users/configs#${config})`,
+    }),
+  );
+
+  return links.length === 1
+    ? links
+    : [
+        links[0],
+        {
+          type: 'text',
+          value: ' and ',
+        } as mdast.Text,
+        links[1],
+      ];
+}
+
+function getRuleDefaultOptions(page: RuleDocsPage): string {
+  const defaults =
+    SPECIAL_CASE_DEFAULTS.get(page.file.stem) ??
+    JSON.stringify(page.rule.defaultOptions);
+
+  const recommended = page.rule.meta.docs.recommended;
+
+  return typeof recommended === 'object'
+    ? [
+        `const defaultOptionsRecommended: Options = ${defaults};`,
+        '',
+        '// These options are merged on top of the recommended defaults',
+        `const defaultOptionsStrict: Options = ${JSON.stringify(recommended.strict)};`,
```

Comment:
I'd have liked to apply the actual merging logic so folks can see what the true defaults are for strict. But that'll be a little more involved. In the interest of getting this done faster I'd rather just merge this PR ASAP. Thoughts?